daml-lf/language/daml-lf.bzl

```diff
@@ -18,16 +18,14 @@
 
 lf_version_configuration = {
     "legacy": "1.8",
+    # TODO (MK) Drop no-exception once Canton fully supports exceptions.
```

Comment:
Have you checked that we don’t use this anywhere else? The reason why I kept it separate is that I wanted to limit the amount of stuff we have to compile & test against 1.13.

Reply:
Yes. 

The only place where it was really used was for the rules:
- `//daml-lf/encoder:testing-dar-lookup-scala-%s` and 
- `//daml-lf/encoder:testing-dar-lookup-lib-%s` and 
The compilation of this two rules induced a minor overhead. 

However, I double check and realized that only `//daml-lf/encoder:testing-dar-lookup-lib-latest` is used, so  deactivate in b2c4e108393ce3b631f70dcef9afca0142ad2552 the compilation for the other keywords, meaning now compile even less.